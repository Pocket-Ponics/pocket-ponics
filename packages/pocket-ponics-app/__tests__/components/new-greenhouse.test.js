import React from 'react'
import { shallow } from 'enzyme'
import { TouchableOpacity } from 'react-native'

import NewGreenhouse from '../../components/greenhouse/new-greenhouse'

describe('New Greenhouse', () => {
	test('renders correctly', () => {
		const tree = shallow(<NewGreenhouse/>).html()
		expect(tree).toMatchSnapshot()
	})

	test('renders correctly', () => {
		const navigation = {
			dispatch: jest.fn()
		}
		const tree = shallow(<NewGreenhouse navigation={navigation}/>)

		tree.find(TouchableOpacity).at(0).props().onPress()
		expect(navigation.dispatch).toHaveBeenCalled()
	})
})