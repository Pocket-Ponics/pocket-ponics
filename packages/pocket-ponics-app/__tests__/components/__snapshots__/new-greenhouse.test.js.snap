// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Greenhouse renders correctly 1`] = `"<View style=\\"color:#FFFFFF;padding:20px;flex-direction:column;flex:1;width:750px\\"><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:20px\\">Begin greenhouse setup</Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Let&#x27;s get your new Pocket &#x27;Ponics greenhouse registered and ready to use. To start, plug your greenhouse into the wall.</Text><Image source=\\"1\\" style=\\"resize-mode:contain;width:100%;flex:1;margin-bottom:20px\\"></Image><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Continue Setup</Text></View><View style=\\"align-items:center;opacity:1\\"><Text style=\\"color:#84AB71;font-size:22px\\">Cancel Setup</Text></View></View>"`;
